apps/src/codebridge/utils/validateFileName.ts

```diff
@@ -0,0 +1,55 @@
+import {ProjectType, FolderId, ProjectFile} from '@codebridge/types';
+
+import codebridgeI18n from '@cdo/apps/codebridge/locale';
+
+import {isDuplicateFileName, DuplicateFileError} from './isDuplicateFileName';
+import {isValidFileName} from './isValidFileName';
+
+type ValidateFileNameArgs = {
+  fileName: string;
+  folderId: FolderId;
+  projectFiles: ProjectType['files'];
+  isStartMode: boolean;
+  validationFile: ProjectFile | undefined;
+};
+
+export const validateFileName = ({
```

Comment:
this and `validateFolderName` should have comments explaining they return an error message (or undefined if no error?)

Reply:
tsdoc added for both this and `validateFileName`